packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -998,19 +998,17 @@ export class ColorPicker
               </div>
               {savedColors.length > 0 ? (
                 <div class={CSS.savedColors}>
-                  {[
-                    ...savedColors.map((color) => (
```

Comment:
was the a reason for `[... ]` here or was it an accident?
git blame shows this was added 4 years ago in the very first version of `calcite-color` component